docs/release-notes/release-1-1-2.md

```diff
@@ -0,0 +1,61 @@
+# Release Notes - ISLE v.1.1.2, 2019-06-27
+
+### Contributions to this release from: 
+
+* David Keiser-Clark (Williams College), documentation
+* Francesca Livermore (Wesleyan University), documentation
+* Gavin Morris (Born-Digital), code updates and testing
+* Mark Sanford (Colgate University), documentation
+* Bethany Seeger (Amherst College), release manager
+* Noah Smith (Born-Digital), code updates and testing
+* Shaun Trujillo (Mount Holyoke College), code updates
+
+### ISLE (main project)
+
+* Documentation for ISLE is now found in the `docs` directory of the ISLE repository
+
+* `docker-compose.yml`
+  * Release header changed to `1.1.2` with date of release
+  * All ISLE docker image tags changed to `1.1.2`
+  
+* Switched to using OpenJDK for Java because of Oracle license changes. This has resulted in a few changes to the docker images:
+  * `isle-ubuntu-basebox` is removed from ISLE, replaced with [adoptopenjdk:openjdk8 image](https://hub.docker.com/r/adoptopenjdk/openjdk8), whose base image is `ubuntu:18.04` (ubuntu-bionic).  This means the underlying operating system setup is the same.
+  * Tools pulled in from the `isle-ubuntu-basebox` and needed by child images have been moved to `isle-tomcat` and `isle-apache` as appropritate.
+  * Djatoka is removed from `isle-imageservices`.  
+  
+* Installer script updates/changes:
+  * OpenSeadragon vset was updated and made more complete for mobile device usage.
+  * New vsets in the installer script to turn off Kakadu by default and to use Imagemagick with OpenJPG instead for JP2 derivative creation.  Please read more on Kakadu licensing on the [Cantaloupe website](https://cantaloupe-project.github.io/manual/4.1/processors.html#KakaduNativeProcessor).
```

Comment:
Am I mis-remembering or will this have the effect of generating JP2 files about 10x bigger now?  If so, that should probably be noted.